quantumdb-core/src/main/java/io/quantumdb/core/schema/definitions/Table.java

```diff
@@ -101,7 +101,7 @@ else if (parent != null && this.parent == null) {
 
 	public Table addIndex(Index index) {
 		checkArgument(index != null, "You must specify an 'index'.");
-		checkState(!containsIndex(index.getIndexName()), "Table already contains an index with name: " + index.getIndexName());
+		checkState(!containsIndex(index.getColumns()), "Table already contains an index: " + index.getIndexName() + " with columns: " + index.getColumns());
```

Comment:
containsIndex only accepts columns, and you do not want the same index (name)

Reply:
But the name isn't based on the columns per se. You could have multiple indices on the same columns, but with different names right?